sdk/openai/Azure.AI.OpenAI/src/Custom/OpenAIClient.cs

```diff
@@ -17,40 +17,256 @@ namespace Azure.AI.OpenAI
     /// <summary> Azure OpenAI APIs for completions and search. </summary>
     public partial class OpenAIClient
     {
+        private const string publicOpenAIVersion = "1";
+        private const string publicOpenAIEndpoint = $"https://api.openai.com/v{publicOpenAIVersion}";
+
+        /// <remarks> This is the deployment id for all Azure OpenAI operations (completions and embeddings) </remarks>
+        private string DeploymentId { get; }
+
+        /// <remarks> This key is used to connect to a non-Azure OpenAI resource created directly with OpenAI.
```

Comment:
nit: These `<remarks>` are on a private member, which means they won't be visible to users via the public documentation or Intellisense, but it sounds like it could be very helpful if they were. What do you think about moving these `<remarks>` to the two constructors that take an `openAIAuthToken`?

Reply:
Agreed, will update.